src/main/java/net/mcreator/blockly/Dependency.java

```diff
@@ -81,6 +81,8 @@ public Color getColor() {
 			return new Color(0x997360).brighter();
 		case "advancement":
 			return new Color(0x68712E).brighter();
+		case "projectileentity":

```

Comment:
This mapping is probably no longer needed with new immediatesourceentity

Reply:
Yeah you're right, sorry I forgot it while removing the mappings. It's now fixed.